Kibana start error

After building and starting the container I only get "Unable to connect" when visiting {dockerhost}:5601

If I start the container without -d and watch the messages I get this:

```
Starting Elasticsearch Server:sysctl: setting key "vm.max_map_count": Read-only file system
.
{"name":"Kibana","hostname":"e8765268b04f","pid":51,"level":50,"err":{"message":"unknown error","name":"Error","stack":"Error: unknown error\n    at respond (/opt/kibana-4.1.1-linux-x64/src/node_modules/elastics
earch/src/lib/transport.js:237:15)\n    at checkRespForFailure (/opt/kibana-4.1.1-linux-x64/src/node_modules/elasticsearch/src/lib/transport.js:203:7)\n    at HttpConnector.<anonymous> (/opt/kibana-4.1.1-linux-x
64/src/node_modules/elasticsearch/src/lib/connectors/http.js:156:7)\n    at IncomingMessage.bound (/opt/kibana-4.1.1-linux-x64/src/node_modules/elasticsearch/node_modules/lodash-node/modern/internals/baseBind.js
:56:17)\n    at IncomingMessage.emit (events.js:117:20)\n    at _stream_readable.js:944:16\n    at process._tickCallback (node.js:442:13)"},"msg":"","time":"2015-11-17T16:54:02.554Z","v":0}
{"name":"Kibana","hostname":"e8765268b04f","pid":51,"level":60,"err":{"message":"unknown error","name":"Error","stack":"Error: unknown error\n    at respond (/opt/kibana-4.1.1-linux-x64/src/node_modules/elastics
earch/src/lib/transport.js:237:15)\n    at checkRespForFailure (/opt/kibana-4.1.1-linux-x64/src/node_modules/elasticsearch/src/lib/transport.js:203:7)\n    at HttpConnector.<anonymous> (/opt/kibana-4.1.1-linux-x
64/src/node_modules/elasticsearch/src/lib/connectors/http.js:156:7)\n    at IncomingMessage.bound (/opt/kibana-4.1.1-linux-x64/src/node_modules/elasticsearch/node_modules/lodash-node/modern/internals/baseBind.js
:56:17)\n    at IncomingMessage.emit (events.js:117:20)\n    at _stream_readable.js:944:16\n    at process._tickCallback (node.js:442:13)"},"msg":"","time":"2015-11-17T16:54:02.563Z","v":0}
```

Followed by stanza after stanza of:

```
Nov 17, 2015 4:54:16 PM org.elasticsearch.transport.netty.MessageChannelHandler messageReceived
WARNING: [logstash-e8765268b04f-52-13876] Message not fully read (response) for [0] handler org.elasticsearch.discovery.zen.ping.unicast.UnicastZenPing$4@4a237b79, error [true], resetting
Nov 17, 2015 4:54:17 PM org.elasticsearch.discovery.zen.ping.unicast.UnicastZenPing$4 handleException
WARNING: [logstash-e8765268b04f-52-13876] failed to send ping to [[#zen_unicast_1#][e8765268b04f][inet[/127.0.0.1:9300]]]
org.elasticsearch.transport.RemoteTransportException: Failed to deserialize exception response from stream
Caused by: org.elasticsearch.transport.TransportSerializationException: Failed to deserialize exception response from stream
        at org.elasticsearch.transport.netty.MessageChannelHandler.handlerResponseError(MessageChannelHandler.java:178)
        at org.elasticsearch.transport.netty.MessageChannelHandler.messageReceived(MessageChannelHandler.java:130)
        at org.elasticsearch.common.netty.channel.SimpleChannelUpstreamHandler.handleUpstream(SimpleChannelUpstreamHandler.java:70)
        at org.elasticsearch.common.netty.channel.DefaultChannelPipeline.sendUpstream(DefaultChannelPipeline.java:564)
        at org.elasticsearch.common.netty.channel.DefaultChannelPipeline$DefaultChannelHandlerContext.sendUpstream(DefaultChannelPipeline.java:791)
        at org.elasticsearch.common.netty.channel.Channels.fireMessageReceived(Channels.java:296)
        at org.elasticsearch.common.netty.handler.codec.frame.FrameDecoder.unfoldAndFireMessageReceived(FrameDecoder.java:462)
        at org.elasticsearch.common.netty.handler.codec.frame.FrameDecoder.callDecode(FrameDecoder.java:443)
        at org.elasticsearch.common.netty.handler.codec.frame.FrameDecoder.messageReceived(FrameDecoder.java:303)
        at org.elasticsearch.common.netty.channel.SimpleChannelUpstreamHandler.handleUpstream(SimpleChannelUpstreamHandler.java:70)
        at org.elasticsearch.common.netty.channel.DefaultChannelPipeline.sendUpstream(DefaultChannelPipeline.java:564)
        at org.elasticsearch.common.netty.channel.DefaultChannelPipeline.sendUpstream(DefaultChannelPipeline.java:559)
        at org.elasticsearch.common.netty.channel.Channels.fireMessageReceived(Channels.java:268)
        at org.elasticsearch.common.netty.channel.Channels.fireMessageReceived(Channels.java:255)
        at org.elasticsearch.common.netty.channel.socket.nio.NioWorker.read(NioWorker.java:88)
        at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioWorker.process(AbstractNioWorker.java:108)
        at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:337)
        at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioWorker.run(AbstractNioWorker.java:89)
        at org.elasticsearch.common.netty.channel.socket.nio.NioWorker.run(NioWorker.java:178)
        at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
        at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
        at java.lang.Thread.run(Thread.java:745)
Caused by: java.io.StreamCorruptedException: Unsupported version: 1
        at org.elasticsearch.common.io.ThrowableObjectInputStream.readStreamHeader(ThrowableObjectInputStream.java:52)
        at java.io.ObjectInputStream.<init>(ObjectInputStream.java:299)
        at org.elasticsearch.common.io.ThrowableObjectInputStream.<init>(ThrowableObjectInputStream.java:44)
        at org.elasticsearch.transport.netty.MessageChannelHandler.handlerResponseError(MessageChannelHandler.java:175)
        ... 23 more
```

Running this on a CentOS 7 VM
Docker version 1.9.0, build 76d6bc9

--- a/Dockerfile
+++ b/Dockerfile
@@ -9,9 +9,9 @@ RUN apt-get -y -qq install wget tar gzip logstash
 
 # Installing Kibana
 RUN mkdir /usr/kibana
-RUN wget -q https://download.elastic.co/kibana/kibana/kibana-4.1.1-linux-x64.tar.gz
-RUN tar -xzf kibana-4.1.1-linux-x64.tar.gz -C /opt
-RUN ln -s /opt/kibana-4.1.1-linux-x64 /opt/kibana
+RUN wget -q https://download.elastic.co/kibana/kibana/kibana-4.2.1-linux-x64.tar.gz
+RUN tar -xzf kibana-4.2.1-linux-x64.tar.gz -C /opt
+RUN ln -s /opt/kibana-4.2.1-linux-x64 /opt/kibana
 
 ADD $PWD/config/setup.sh /tmp/setup.sh
 RUN chmod +x /tmp/setup.sh

--- a/config/logstash.conf
+++ b/config/logstash.conf
@@ -75,9 +75,6 @@ filter {
 }
 
 output {
-        stdout { codec => "rubydebug" }
-
-        elasticsearch {
-                  host => '127.0.0.1'
-        }
+  elasticsearch { host => ["localhost:9200"] }
+  stdout { codec => rubydebug }
 }

